helix-core/src/main/java/org/apache/helix/util/StatusUpdateUtil.java

```diff
@@ -406,6 +406,15 @@ public void logError(Message message, Class classInfo, Exception e, String addit
         additionalInfo + sw.toString(), manager);
   }
 
+  public void logError(Message message, Class classInfo, Throwable t, String additionalInfo,
```

Comment:
this may not be required. let us use existing one where the additional info as 'T' info.